content/blog/teach-tidyverse-2021/index.Rmd

```diff
@@ -0,0 +1,437 @@
+---
+output: hugodown::hugo_document
+
+slug: teach-tidyverse-2021
+title: Teaching the tidyverse in 2021
+date: 2021-08-31
+author: Mine Çetinkaya-Rundel
+description: >
+    Recommendations for teaching the tidyverse in 2021, summarizing package updates most relevant for teaching data science with the tidyverse, particularly to new learners.
```

Comment:
Very small, but this needs a line break — since you're using the yaml `>` you can spread across multiple lines with out worrying.